src/PackageManager/DynamoPackageManagerExtension_ViewExtensionDefinition.xml

```diff
@@ -0,0 +1,4 @@
+<ViewExtensionDefinition>
+  <AssemblyPath>..\PackageManager.dll</AssemblyPath>
```

Comment:
Should we call this assembly as DynamoPackagesUI.dll just to tell that it's related to UI only? We also have DynamoPackages as an extension.